docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,700 @@
+    publicKey,
+) { result ->
+    when (result) {
+        is Error -> ...
+        is Success -> ...
+    }
+}
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+try await UID2Manager.shared.generateIdentity(
+    .phoneHash("eVvLS/Vg+YZ6+z3i0NOpSXYyQAfEXqCZ7BTpAjFUBUc="),
+    subscriptionID: subscriptionID,
+    serverPublicKey: serverPublicKeyString
+) 
+```
+
+</TabItem>
+</Tabs>
+
+In this scenario: 
+
+- The publisher is responsible for normalizing and hashing the phone number. For details, see [Phone Number Hash Encoding](../getting-started/gs-normalization-encoding.md#phone-number-hash-encoding).
+- The UID2 mobile SDK encrypts the hashed DII before sending it to the UID2 service.
+
+## Token Storage and Refresh
+
+After a call to the applicable method listed in [Configure the UID2 Mobile SDK](#configure-the-uid2-mobile-sdk) is successful, an identity is generated and stored in local file storage. The UID2 mobile SDK refreshes the UID2 token periodically. 
+
+:::warning
+The format of the file stored in the local file storage, or the filename itself, could change without notice. We recommend that you do not read and update the file directly.
+:::
+ 
+## Pass Generated Token for Bid Stream Use
+
+In your mobile app, if the call to `generateIdentity()` was successful, it returned an identity, which you can then use in the bid stream.
+
+First, the method call to generate the identity:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().generateIdentity()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.generateIdentity()
+```
+
+</TabItem>
+</Tabs>
+
+Then, the method call to get the advertising token:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+UID2Manager.getInstance().getAdvertisingToken()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+UID2Manager.shared.getAdvertisingToken()
+```
+
+</TabItem>
+</Tabs>
+
+If successful, this method call returns a non-null string object such as the following: 
+
+```js
+AgAAAQFt3aNLXKXEyWS8Tpezcymk1Acv3n+ClOHLdAgqR0kt0Y+pQWSOVaW0tsKZI4FOv9K/rZH9+c4lpm2DBpmFJqjdF6FAaAzva5vxDIX/67UOspsYtiwxH73zU7Fj8PhVf1JcpsxUHRHzuk3vHF+ODrM13A8NAVlO1p0Wkb+cccIIhQ==
+```
+
+You can use this identity to pass downstream for sending in the RTB bid stream.
+
+If the `getAdvertisingToken()` method call returns `null`, there was no identity or valid token generated.
+
+Some possible reasons for this, and some things you could do to troubleshoot, are as follows:
+
+- Check to see whether there are any errors from the `generateIdentity()` call.
+- Enable logging to get more information: see [Enable Logging](#enable-logging)
+- It's possible that the identity that was generated has expired. If this happens, you'll need to call the `generateIdentity()` method again: see [Client-Side Integration Example](#client-side-integration-example).
```

Comment:
the link should be to the "Configure the UID2 Mobile SDK" section which explains call generateIdentity method

Reply:
Done, thx.